src/cpu/core_dynrec/Makefile

```diff
@@ -136,7 +136,7 @@ INSTALL_SCRIPT = ${INSTALL}
 INSTALL_STRIP_PROGRAM = $(install_sh) -c -s
 LDFLAGS = 
 LIBOBJS = 
-LIBS = -lasound -lm -ldl -lpthread -L/usr/lib -Wl,-rpath,/usr/lib -lSDL -lncurses -lpng -lz -lpcap -lSDL_net -lX11 -lfluidsynth
+LIBS = -lasound -lm -ldl -lpthread -L/usr/lib -Wl,-rpath,/usr/lib -lSDL -lncurses -lpng -lz -lpcap -lslirp -lSDL_net -lX11 -lfluidsynth
```

Comment:
Hopefully not needed?

We've got `LIBS=` assigning the library as provided by PKG_CONFIG:

`LIBS="$LIBS ${SLIRP_LIBS}` or
`LIBS="$LIBS "`pkg-config slirp --libs`

So this feels a bit fragile; unless this is for a non-autotools manages build?